docker/bin/run.sh

```diff
@@ -25,10 +25,10 @@ if [ ! -f /var/www/html/wp-config.php ]; then
 	while [ "$i" -le "$times" ]; do
 		sleep 3
 		wp --allow-root config create \
-			--dbhost=$WORDPRESS_DB_HOST \
-			--dbname=wordpress \
-			--dbuser=$WORDPRESS_DB_USER \
-			--dbpass=$WORDPRESS_DB_PASSWORD \
+			--dbhost=${DB_HOST} \
+			--dbname=${DB_NAME} \
+			--dbuser=${DB_USER} \
+			--dbpass=${DB_PASSWORD} \
```

Comment:
Let's leave these env variables named the same please. Like `WORDPRESS_DB_USER` and such.

Reply:
Yup, I don't have a strong opinion on this. :-) My initial suggestion was to use these:
```
WP_DOMAIN
WP_ADMIN_USER
WP_ADMIN_EMAIL
WP_ADMIN_PASSWORD
WP_TITLE
DB_NAME
DB_USER
DB_PASSWORD
DB_ROOT_PASSWORD
SFTP_USER
SFTP_PASSWORD
```

Mostly just because kind of everything would be `WP_*` (or `WORDPRESS_*` like initially).

Some other vars we might add in future:
- WP_VERSION
- WP_CLI_CONFIG_PATH
- PHP_VERSION
- PHPUNIT_VERSION
- Some xdebug specific?

Since people will copy these over to their `.env` files, changing these now can still happen — later it'll be too much of a hassle.

Thoughts?

Reply:
Actually, I figured something unique how we need to configure (and thus name) these variables, because we don't store `.env` file in the same folder where we run `docker-compose` command. I'll update and elaborate more in the issue description.